wd_demo.au3

```diff
@@ -817,20 +817,26 @@ Func DemoSelectOptions()
 	MsgBox($MB_OK + $MB_TOPMOST + $MB_ICONINFORMATION, "Information", "After DESELECTALL")
 
 	; select desired options
-	Local $aOptions[] = ['Cat', 'Hamster', 'Parrot', 'Albatross']
-	_WD_ElementSelectAction($sSession, $sSelectElement, 'MULTISELECT', $aOptions)
+	Local $aOptionsToSelect[] = ['Cat', 'Hamster', 'Parrot', 'Albatross']
+	_WD_ElementSelectAction($sSession, $sSelectElement, 'MULTISELECT', $aOptionsToSelect)
 	If @error Then Return SetError(@error, @extended, '')
 	MsgBox($MB_OK + $MB_TOPMOST + $MB_ICONINFORMATION, "Information", "After MULTISELECT: Cat / Hamster / Parrot / Albatros")
 
+	; retrieves all <option> elements as 2D array
```

Comment:
This should be "selected", not "all"

Reply:
Thanks, fixed: [comment fix](https://github.com/Danp2/au3WebDriver/pull/367/commits/a4b28372f3b563d96f98a929a7cacf1dff6ef2cb)